Core/Authenticator.php

```diff
@@ -16,7 +16,7 @@ public function attempt($email, $password)
         $user = App::resolve(Database::class)->query('select * from users where email = :email', [
             'email' => $email,
```

Comment:
you could check if the user is deleted in the query.
```suggestion
            'email' => $email,
            'is_deleted' => 0,
```